onnxruntime/test/providers/tensorrt/tensorrt_basic_test.cc

```diff
@@ -87,6 +88,97 @@ void CreateBaseModel(std::string model_name, std::string graph_name, std::vector
   status = onnxruntime::Model::Save(model, model_name);
 }
 
+void RunInference(std::string model_name, std::string sess_log_id) {
```

Comment:
i think we need to clarify with the naming that each thread is creating its own session and only 1 thread is running inference per session.
we need a multiple run, single session test as well.

Reply:
Rename the function.